src/item.cpp

```diff
@@ -3786,6 +3786,9 @@ int item::spoilage_sort_order()
     }
 
     if( subject->goes_bad() ) {
+        if( subject -> is_corpse() ) {
+            return to_turns<int>( 14400_turns - subject->rot );
```

Comment:
I'd consider putting this `14400_turns` value in `game_constants.h` as `CORPSE_ROT_TIME` or something similar.